### PR TITLE
fix: dead link

### PR DESCRIPTION
**Pull request by ldicarlo**
### 🤔 What's changed?

It's a dead link in the documentation.

### ⚡️ What's your motivation? 

I wanted the link to point to the right place

### 📋 Checklist:

- [x] I agree to respect and uphold the [Cucumber Community Code of Conduct](https://cucumber.io/conduct/)

### CODE REVIEW

File: content/docs/cucumber/debugging.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ In order to debug your scenarios on the JVM, you can step through the the steps
 
 1. Set a breakpoint on the part of the code you want to debug. This might be the line you are currently getting an Exception (see your stacktrace). 
 
-2. Run your [RunCucumberTest](https://github.com/cucumber/cucumber-java-skeleton/blob/master/src/test/java/io/cucumber/skeleton/RunCucumberTest.java) in debug mode
+2. Run your [RunCucumberTest](https://github.com/cucumber/cucumber-java-skeleton/blob/maven/master/src/test/java/io/cucumber/skeleton/RunCucumberTest.java) in debug mode
```
Comment (luke-hill) [inline content/docs/cucumber/debugging.md]: Think the url you want is this: https://github.com/cucumber/cucumber-java-skeleton/blob/main/maven/src/test/java/io/cucumber/skeleton/RunCucumberTest.java

**Comment by luke-hill** [comment]
Hi @ldicarlo are you able to fix this link?

File: content/docs/cucumber/debugging.md (hunk 1 above)
Comment (ldicarlo) [reply content/docs/cucumber/debugging.md]: Yes you are right. This is strange though because I thought I tested it when doing the PR

**Comment by ldicarlo** [comment]
> Hi @ldicarlo are you able to fix this link?

It is updated. Sorry, I did not receive any notification before.

**Review by luke-hill — APPROVED** [review]

**Comment by aslakhellesoy** [comment]
Hi @ldicarlo,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Pull request merged**